source/data/tier1data/features/group_administration.rst

```diff
@@ -0,0 +1,125 @@
+Alternatively, they can go straight to *https://mango.vscentrum.be/data-platform/project/<project_name>*.
+
+On this page, you get an overview of all members in the project:
+
+.. figure:: ../images/group_administration/mango_portal_manage_members.png
+  :alt: An overview of all members.
+
+- To add a new member to the project, click on 'add member' and search them by their name or account number. Next, click on 'Add'.
+
+- To change the role of a member, click on the blue pencil icon next to their name. Select their new role on the dropdown, and click on 'Update'.
+
+- To remove a member from the project, click on the red trashcan icon next to their name. In the menu that pops up, click on 'Delete'.
+
+After making one or multiple of these changes, a button 'apply changes in iRODS' will appear.
+Click on this button to finalize your changes. 
+
+
+
+******************
+Managing subgroups
+******************
+
+Subgroups are used to gives permissions to a specific group of members from your project.
+Managers can create and manage subgroups by going to the Group Administration tab of the ManGO portal.
+
+After selecting your project, you will see an overview of all subgroups. 
+If they have a lock sign next to them, these are managed by the platform and can be viewed, but not edited.
+Examples include:
+
+- The main group of your project  
+- The 'managers' group  
+- ...
+
+Other groups, which you can create yourself, are called 'custom groups'.
+For example, you could create a custom group for staff and one for master students. 
+To do so, click on 'Add group' and give a name.
+Custom groups will always get your project name as prefix. For example, if your project is called 'demo-project' and you enter 'master_students', the custom group will be called 'demo-project_master_students'.
+
+When you create a new group, or where you click on one, you will get an overview of the group members on the left side.
+On the right side, you will see the members of the project who are not yet part of this group.  
+
+
+.. figure:: ../images/group_administration/mango_portal_group_administration.png
+  :alt: Editing the members of a custom group
+
+To add a member to the group, select their checkbox and click on 'Add selected users'.
+
+Likewise, to remove users from the group, select their checkbox and click on 'Remove selected users'.
+
+**Note:** You may need to log out and login again before you see changes you made in groups reflected when editing permissions. 
+
+
+
+Help! I want to collaborate with an external researcher
+-------------------------------------------------------
+
+As demonstrated earlier, you can add collaborators to your group if they have a VSC account
+If you want to work together with someone who doesn't have a VSC account, there are two options:
+
+1) Applying for a VSC account
+
+You can let your collaborators apply for a VSC account.

```

Comment:
This is not always possible. I would rephrase this to something like:

> If your collaborators are eligible for a VSC account, we recommend that they apply for one to access Tier-1 data.

Reply:
Thanks, I didn't realize that. I will phrase it as 'they can' instead of 'we recommend' (because we recommend either a VSC account or Globus depending on the context).